Module.php

```diff
@@ -16,6 +17,10 @@ public static function showBreakingNews()
             return false;
         }
 
+        if (Yii::$app->controller instanceof MustChangePasswordController) {
```

Comment:
Can we somehow detect the "login" layout? This would catch more cases like this.

Reply:
@luke- Done in commit https://github.com/humhub-contrib/breakingnews/pull/27/commits/b3ff34c9e041c016a04edd7ca05d5febfa07589b.
I find the layout `@humhub/modules/user/views/layouts/main` is used for login, password recovery and etc. controllers.